.github/workflows/build_and_test.yml

```diff
@@ -4587,7 +4587,6 @@ jobs:
       shell: bash
       run: |
         cd OpenDDS/ACE_TAO
-        perl -ni.bak -e "print unless /opendds/" ACE/bin/MakeProjectCreator/config/default.features # remove opendds features from here, let individual build configure scripts handle those
```

Comment:
It seems like it would make more sense to tweak the downloaded ACE_TAO artifact's `default.features` to include the desired opendds features before running configure, since theoretically more than one build job could use this ACE_TAO artifact.

Since it's just one build, I agree it's not a big deal at the moment, but you'll need to make sure you've added the features to the configure line for the ACE_TAO call to configure (perhaps they are already there) and then bump the cache key index value.

Reply:
Many ACE_TAO jobs are using this line to remove features with `opendds` in their names. So we may want to check that each OpenDDS build job has all the intended features which can be missing in `default.features`.